transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -37,17 +37,20 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
 
     // read in extracted info
     val donorInputs = readRawEntities(EncodeEntity.Donor)
+    val antibodyInputs = readRawEntities(EncodeEntity.AntibodyLot)
     val fileInputs = readRawEntities(EncodeEntity.File)
 
     val donorOutput = donorInputs
       .withName("Transform Donor objects")
       .map(DonorTransformations.transformDonor)
+    val antibodyOutput = antibodyInputs.map(AntibodyTransformations.transformAntibody)
 
     // write back to storage
+    StorageIO.writeJsonLists(donorOutput, "Donors", s"${args.outputPrefix}/donor")
     StorageIO.writeJsonLists(
-      donorOutput,
-      "Donors",
-      s"${args.outputPrefix}/donor"
+      antibodyOutput,
+      "Antibodies",
+      s"${args.outputPrefix}/antibody"
```

Comment:
Smallest nit/question about style - is it preferable to have the params for a function all on one line vs on separate new lines? I noticed my IntelliJ likes to spread things out but it doesn't do it all the time so some similar calls in my code are formatted differently just like this.

Reply:
Ope, sorry. I merged before I saw your comment. I think the compiler reformats things when it runs, but I don't know all of the rules. It seems like this antibody output command exceeds the line width limit so it reformats it, but the donor command is slightly shorter.

Reply:
It would probably be worth making them all use the same format though, I agree.